### DISCUSSION TITLE
Modern Polars - Summary

### DISCUSSION THREAD

**Discussion by giscus** [kevinheavey/modern-polars#3] [General]
# Modern Polars - Summary

https://kevinheavey.github.io/modern-polars/summary.html

**Comment by AlFontal** [comment]
Hey!
I just wanted to say this was a great write-up.
While I've been hearing of `polars` for a while I have always been rather reluctant given how comfortable and fluent I feel with the `pandas` API, no matter the oddities.

You have inspired me to go ahead and try it. I don't think I am going to change any working codebases but my next project is going to use `polars` for sure. I think the userbase is growing rather quickly and as the package matures I expect the little things it doesn't seem to do well yet will be solved. 

In any case, I think that competition is really good for the python ecosystem.

**Comment by joun1128** [comment]
Thank you for editing good reference.
It was very helpful to try and get a sense of polars.

**Comment by ds-ramakant** [comment]
Fantastic write up. Highly educational for me who's not a coder and find the tidyverse approach far more easier to read and understand. While I'm trying to learn python the learning curve is still uphill for me. Polars looks like a good alternative to experiment with

**Comment by Cacunas1** [comment]
Tahnks a lot for this guide. I was in need of 1:1 examples with pandas to make the migration smoother. IDK if ti's too much to ask, but could you think about writting about interactions with DBs? I'm currently reading/writing to a PostgreSQL DB, and have to resort to converting to pandas to do the usual stuff. I know that ```write_database()``` was recently added, but I ran into a weird exception while trying to use it. And I'm very curios into seeing how ti works in conjunction with ADBC, but I haven't found any examples of being used with polars. Anyways, thanks a lot!
